itest/rust/src/object_tests/dyn_gd_test.rs

```diff
@@ -445,3 +445,23 @@ impl InstanceIdProvider for foreign::NodeHealth {
         self.base().instance_id()
     }
 }
+
+// ----------------------------------------------------------------------------------------------------------------------------------------------
+// Check if DynGd can be properly exported
+
+#[derive(GodotClass)]
+#[class(init)]
+struct RcDynGdExporter {
+    #[var]
+    only_node_health: Option<DynGd<foreign::NodeHealth, dyn Health>>,
+}
```

Comment:
What does "Rc" stand for here?

If you mean "RefCounted", our typical abbreviation is "Refc" to distinguish from Rust's standard `Rc` pointer.

Also, do we need two separate exporter classes here, or can we just have one more field in `DynGdExporter`? Every class added to tests adds permanent runtime overhead in CI, which may add up over time...

Reply:
`RcDynGdExporter` is Refcounted which makes it eligible to check for failing tests (since it will be properly cleaned up without much issue).

There is another way though – I'll move these tests to already existing test suite (the one with HasProperty)

Reply:
So you _do_ need 2 classes `RefcDynGdExporter` and `DynGdExporter`? If yes, then keep them.

No need to create huge non-locality, it's better to keep this in the `dyn_gd_test.rs`  file if possible, otherwise tests can become quite hard to follow. I just thought it would be easy to merge the two, but restructuring unrelated parts of the tests isn't worth it.

Reply:
After some consideration - one test class (now named `RefcDynGdExporter`)  should be enough.